library/core/src/ops/bit.rs

```diff
@@ -668,6 +669,7 @@ shr_impl_all! { u8 u16 u32 u64 u128 usize i8 i16 i32 i64 i128 isize }
 /// assert_eq!(bv, expected);
 /// ```
 #[lang = "bitand_assign"]
+#[doc(alias = "&")]
```

Comment:
Putting an alias for the non-assignment operator on the assigning flavor would violate this part of the std doc policy

> There must be an obvious single target for the alias that is an exact analogue of the aliased name. We will not add the same alias to multiple declarations. (const and non-const versions of the same function are fine.) We will also not add an alias for a function that's only somewhat similar or related.

https://std-dev-guide.rust-lang.org/documentation/doc-alias-policy.html

Reply:
I doubt that those guidelines do fully apply too *symbolic* aliases. I.e. aliases that aren't an identifier. I was also following a convention that seems to already exist for several operators like `+=`, `-=`, `*=`, and `/=`, e. g.https://doc.rust-lang.org/std/?search=%2B

Reply:
It may be the case that those aliases should be removed rather than be taken as precedent. Especially if they only exist to work around a search limitation (see other comment thread).

Reply:
Alright… what’s the proper order to do that in then? Remove the other aliases now? Or wait until search is improved?

Reply:
IMO, this whole PR should be closed, rustdoc should be changed to handle fuzzy search, and then we can remove most of the remaining symbolic aliases.

Reply:
> IMO, this whole PR should be closed

I’d want to keep 49c680ada001a4c296d1c00f58be8d8b535d50f6, ab89c88faadbc5a4063c8473c9ce28af16324d79, 56634a1ef35a591aa735e883e42c976cf25cffdc, and part of 0a565985d178b3fa70af26f4fb3c8719b6e10f7e though.

For 0a565985d178b3fa70af26f4fb3c8719b6e10f7e I’d want to keep `*`, `*const`, and `*mut` for now with the intention of `*` being removed at a later time. Though the inclusion of `*` can be debated.